packages/x-date-pickers/src/locales/ptBR.ts

```diff
@@ -0,0 +1,55 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+import { CalendarPickerView } from '../internals/models';
+
+const ptBRPickers: Partial<PickersLocaleText<any>> = {
+  // Calendar navigation
+  previousMonth: 'Mês anterior',
+  nextMonth: 'Próximo mês',
+
+  // View navigation
+  openPreviousView: 'Abrir próxima seleção',
+  openNextView: 'Abrir seleção anterior',
+
+  calendarViewSwitchingButtonAriaLabel: (view: CalendarPickerView) =>
+    view === 'year'
+      ? 'Seleção de ano está aberta, alternando para seleção de calendário'
+      : 'Seleção de calendários está aberta, alternando para seleção de ano',
+
+
+  // DateRange placeholders
+  start: 'Início',
+  end: 'Fim',
+
+  // Action bar
+  cancelButtonLabel: 'Cancelar',
+  clearButtonLabel: 'Limpar',
+  okButtonLabel: 'OK',
+  todayButtonLabel: 'Hoje',
+
+  // Clock labels
+  clockLabelText: (view, time, adapter) =>
+    `Selecione ${view}. ${
+      time === null ? 'Hora não selecionada' : `Selecionado a hora ${adapter.format(time, 'fullTime')}`
```

Comment:
Better this way?

```suggestion
      time === null ? 'Hora não selecionada' : `Hora ${adapter.format(time, 'fullTime')} selecionada`
```

Reply:
Maybe `Hora selecionada: ${adapter.format(time, 'fullTime')} `, but  `Hora ${adapter.format(time, 'fullTime')} selecionada` is not natural. 

 `Selecionado a hora ${adapter.format(time, 'fullTime')}` sounds natual as well

Reply:
I didn't use  `Hora selecionada: ${adapter.format(time, 'fullTime')} ` because any other supported language has this "pattern" with ":"